nfsexports.go

```diff
@@ -136,7 +136,7 @@ func List(exportsFile string) (map[string]string, error) {
 		}
 	}
 
-	if scanner.Err() != nil {
+	if err = scanner.Err(); err != nil {
```

Comment:
You should use := in this case to not use the outer scope err but actually declare a new variable